src/BaGet.Azure/Search/AzureSearchService.cs

```diff
@@ -163,7 +163,11 @@ public class AzureSearchService : ISearchService
 
             var response = await _searchClient.Documents.SearchAsync<PackageDocument>(query, parameters, cancellationToken: cancellationToken);
             var results = response.Results
-                .Select(r => r.Document.Id)
+                .Select(r => new DependentResult() {
```

Comment:
Minor style nitpick:

```suggestion
                .Select(r => new DependentResult
                {
```